webcompat/templates/issue/issue-comment-list.html

```diff
@@ -0,0 +1,24 @@
+{% autoescape false %}{% for response in comments %}
```

Comment:
I'm not sure we want to disable autoescaping... I don't recall why I added this, but removing it doesn't do anything strange AFAICT.